dsaas-services/che-monitoring.yaml

```
@@ -1,5 +1,5 @@
 services:
-- hash: 91889753e52d1054befc8b5ce7c37f879eff5d9d
+- hash: 2a37f1f0b91f677312ce4bcf3b134e151b379147
```

```suggestion
- hash: 779dc32a961a4a748fe627f60f5eebdf1b0d5d9e
```

Lets bump this to the hash that has the path fixes